coverage.out

```diff
@@ -0,0 +1 @@
+mode: set
```

Comment:
This looks like a redundant file! add by accident?

Reply:
ah yes, thanks for pointing it out, file has been removed